web/react/components/setting_item_max.jsx

```diff
@@ -15,7 +15,7 @@ export default class SettingItemMax extends React.Component {
 
         var extraInfo = null;
         if (this.props.extraInfo) {
-            extraInfo = this.props.extraInfo;
+            extraInfo = (<div className='setting-list__hint'>{this.props.extraInfo}</div>);
```

Comment:
Will this break anything that was already passing in extraInfo as a prop without the assumption of being inclosed in the new div?